docs/fidesops/docs/guides/saas_config.md

```diff
@@ -0,0 +1,256 @@
+
+In short, you can think of the Dataset as the "what" (what data is available from this API) and the SaaS config as the "how" (how to access and update the data).
+
+#### An example SaaS config
+
+This config defines:
+
+- The domain and authentication requirements for an HTTP client to Mailchimp
+- A test request for verifying the connection was set up correctly
+- Endpoints to the following resources within the Mailchimp API:
+    - GET and UPDATE for the `members` resource
+    - GET for the `conversations` resource
+    - GET for the `messages` resource
+
+```yaml
+saas_config:
+  fides_key: mailchimp_connector_example
+  name: Mailchimp SaaS Config
+  description: A sample schema representing the Mailchimp connector for Fidesops
+  version: 0.0.1
+
+  connector_params:
+    - name: domain
+    - name: username
+    - name: api_key
+
+  client_config:
+    protocol: https
+    host:
+      connector_param: domain
+    authentication:
+      strategy: basic_authentication
+      configuration:
+        username:
+          connector_param: username
+        password:
+          connector_param: api_key
+
+  test_request:
+    path: /3.0/lists
+    
+  endpoints:
+    - name: messages
+      requests:
+        read:
+          path: /3.0/conversations/<conversation_id>/messages
+          request_params:
+            - name: conversation_id
+              type: path
+              references:
+                - dataset: mailchimp_connector_example
+                  field: conversations.id
+                  direction: from
+          data_path: conversation_messages
+    - name: conversations
+      requests:
+        read:
+          path: /3.0/conversations
+          request_params:
+            - name: count
+              type: query
+              default_value: 1000
+            - name: offset
+              type: query
+              default_value: 0
+            - name: placeholder
+              type: query
+              identity: email
+    - name: member
+      requests:
+        read:
+          path: /3.0/search-members
+          request_params:
+            - name: query
+              type: query
+              identity: email
+          data_path: exact_matches.members
+        update:
+          path: /3.0/lists/<list_id>/members/<subscriber_hash>
+          request_params:
+            - name: list_id
+              type: path
+              references:
+                - dataset: mailchimp_connector_example
+                  field: member.list_id
+                  direction: from
+            - name: subscriber_hash
+              type: path
+              references:
+                - dataset: mailchimp_connector_example
+                  field: member.id
+                  direction: from
+```
+
+
+A SaaS config schema is broken into the following sections:
+
+- Metadata
+- Connector params
+- Client config
+- Test request
+- Endpoints
+
+#### Metadata
+This includes the following fields:
```

Comment:
This is a little confusing because there's no metadata header

Reply:
I can see how this is a bit confusing, I cleaned this up to prevent others from getting the idea there's a metadata header.